### ISSUE TITLE
Communities don't seem to propagate their color properly

### ISSUE THREAD

**Issue by PascalPrecht** [status-im/status-desktop#2826]
One thing I've notice when working on #2805 was that community colors (the ones used in the letter icon) aren't probably available at all times.

For example, initially when starting the app and switching to a community view, the community's channel icons are all in a default color (some blue in latest `master` at the time of creating this issue, but `black` when tested in #2806 ), except for the currently active channel. 

When changing channels, the newly selected one **then** shows its proper color. 

So in other words, it looks like one has to interact with the UI to receive the community color.
The issue is also visible in the top bar component, especially in PR (#2824).

Looks like we're missing some signal emission somewhere.

**Comment by alaibe** [comment]
@PascalPrecht to confirm, is this about the community or about the channel in the community that the problem is.
See below screenshot:
![image.png](https://images.zenhubusercontent.com/55a8a95e4f7a08fbb7f3c6dc/216cf0f6-c9af-4b9e-8286-7228a696d86f)

Or is the problem in the screenshot different than the one you mentioned?

**Comment by PascalPrecht** [comment]
@alaibe that's correct! It's about the channels, sorry for the confusion!

**Comment by PascalPrecht** [comment]
Reopening this, as the issue still persists:

<img width="732" alt="Screenshot 2021-07-20 at 10 34 54" src="https://user-images.githubusercontent.com/445106/126289117-9a14bb1e-2dae-4af8-8cc2-6720e962ba08.png">

/cc @alaibe

**Comment by alaibe** [comment]
New causes: when adding a new channel/update of the community or accepting a membership request
